packages/api/src/services/emp-state.ts

```diff
@@ -155,9 +155,8 @@ export default (config: Config, appState: Dependencies) => {
   }
 
   async function update(startBlock?: number | "latest", endBlock?: number) {
-    await Promise.map(Array.from(await registeredEmps.values()), (address: string) =>
-      updateOne(address, startBlock, endBlock)
-    );
+    const addresses = Array.from(await registeredEmps.values());
+    await Promise.mapSeries(addresses, (address: string) => updateOne(address, startBlock, endBlock));
```

Comment:
Why do we run these in series rather than in parallel?

Reply:
its a soft way to slow down requests, considering theres like 100 emps, i didnt want to blast infura. But maybe that isnt necessary